### PR TITLE
[HOTFIX][ENG-2913][ENG-2917][ENG-2924] Character lab + keen analytics

### PR DESCRIPTION
**Pull request by adlius**
- Ticket: [ENG-2913] [ENG-2917] [ENG-2924]
- Feature flag: n/a

## Purpose
- Character Lab: Make it so moderators can add new registrations https://github.com/CenterForOpenScience/ember-osf-web/pull/1217

- Character Lab: Add new fields for Character Lab ( Study Design and School Type) on Collections Discover page and Collection Submission page. Users should be able to filter by these new fields and see them visible on Collection search cards on the discover page https://github.com/CenterForOpenScience/ember-osf-web/pull/1219

- Keen analytics: Don't load keen twice on load https://github.com/CenterForOpenScience/ember-osf-web/pull/1218

## QA Notes

[ENG-2913]: https://openscience.atlassian.net/browse/ENG-2913
[ENG-2917]: https://openscience.atlassian.net/browse/ENG-2917
[ENG-2924]: https://openscience.atlassian.net/browse/ENG-2924

### CODE REVIEW

File: lib/registries/addon/components/registries-navbar/component.ts [hunk 1]
```diff
@@ -54,7 +54,7 @@ export default class RegistriesNavbar extends AuthBase {
         if (this.provider.id === 'egap') {
             return this.features.isEnabled(camelize(egapAdmins));
         }
-        return this.provider.allowSubmissions;
+        return this.provider.allowSubmissions || this.provider.currentUserCanReview;
```
Comment (fabmiz) [inline lib/registries/addon/components/registries-navbar/component.ts:57]: add `currentUserCanReview` to cp dependent keys

**Review by fabmiz — CHANGES_REQUESTED** [review]
Looks good. Please add a PR description and add/update [tests](https://github.com/CenterForOpenScience/ember-osf-web/blob/master/tests/engines/registries/acceptance/branded/new-test.ts#L31)

File: lib/registries/addon/components/registries-navbar/component.ts [hunk 2]
```diff
@@ -54,7 +54,7 @@ export default class RegistriesNavbar extends AuthBase {
         if (this.provider.id === 'egap') {
```
Comment (fabmiz) [inline lib/registries/addon/components/registries-navbar/component.ts:54]: @brianjgeiger, with these changes, is there any need to keep this `egap` special case?

Comment (brianjgeiger) [reply lib/registries/addon/components/registries-navbar/component.ts:54]: No, egap is now waffled on for everyone in production anyways, so this is ready to go away whenever.

**Review by futa-ikeda — APPROVED** [review]
🚀

**Review by fabmiz — APPROVED** [review]
💯

**Pull request merged**